content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -0,0 +1,169 @@
+---
+Title: Prometheus integration with Redis Cloud
+linkTitle: Prometheus integration
+description: To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud databases.
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+---
+
+To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud subscription.
+
+- [Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting toolkit that can scrape metrics from different sources.
+- [Grafana](https://grafana.com/) is an open-source, feature-rich metrics dashboard and graph editor that can process Prometheus data.
+
+For more information on how Prometheus communicates with Redis Enterprise Software clusters, see [Prometheus integration with Redis Enterprise Software]({{< relref "/rs/clusters/monitoring/prometheus-integration" >}}).
+
+{{< note >}}
+
+You must [enable VPC peering]({{< relref "/rc/security/vpc-peering" >}}) for your subscription to connect Prometheus and Grafana to your Redis Cloud databases. VPC peering is available only with Flexible or Annual subscriptions.  Prometheus and Grafana cannot connect to databases on Fixed or Free subscriptions.
+
+{{< /note >}}
+
+## Quick start
+
+To get started with custom monitoring with Prometheus on Docker:
+
+1. To connect Prometheus and Grafana to your Redis Cloud databases, they must be running on an machine that:
```

Comment:
I agree with Kaitlyn's suggestion to add a prerequisites section. Also, you might want to reword this a bit to avoid starting with "to" twice in a row and also avoid phrases like "be running".
```suggestion
1. To connect Prometheus and Grafana to your Redis Cloud databases, they must be running on a machine that:
```